I do magic/promises-callbacks-js/src/css/grid.scss

```diff
@@ -0,0 +1,143 @@
+@import "_variables.scss";
+
+/*
+.class {
+  @media screen and (min-width: 480px) {}
+  @media screen and (min-width: 768px) {}
+  @media screen and (min-width: 1024px) {}
+  @media screen and (min-width: 1200px) {}
+}
+*/
+
+.menu-button {
+  @media(min-width: 1024px) {
+    display: none;
+  }
+}
+
+.options {
+  @media(min-width: 1024px) {
+    display: flex;
+  }
+}
+
+#agent-section {
+  @media(min-width: 768px) {
+    flex-direction: row;
+    padding: 0;
+    justify-content: space-around;
+    padding-bottom: 50px;
+  }
+}
+
+.agent {
+  @media(min-width: 768px) {
+    width: 348px;
+  }
+  @media(min-width: 1024px) {
+    &__name {
+      left: calc(50% - 160px);
```

Comment:
Is it really necessary to do all these calculations always? Whenever the window is resized, all this calculations are made. Maybe you can use a different measures based on another component instead of the whole window.
Don't need to change it, but to take in mind.

Reply:
Done